src/bootstrap.jl

```diff
@@ -290,7 +304,7 @@ function tidyσs(bsamp::MixedModelBootstrap{T}) where {T}
     )
     for (iter, r) in enumerate(bstr)
         setθ!(bsamp, iter)    # install r.θ in λ
-        σ = r.σ
+        σ = r.σ === missing ? 1 : r.σ
```

Comment:
This could be written as
```
σ = coalesce(r.σ, one(T))
```
to keep type consistency and more in the spirit of the `missing` design.

Reply:
Nice, thanks for the tip!